src/test/java/uk/gov/ons/census/casesvc/service/FulfilmentRequestProcessorTest.java

```diff
@@ -103,4 +169,89 @@ public void testNullDateTime() {
       throw re;
     }
   }
+
+  private void testIndividualResponseCode(String individualResponseCode) {
+    // Given
+    Case parentCase = getRandomCase();
+    parentCase.setUacQidLinks(new ArrayList<>());
+    parentCase.setEvents(new ArrayList<>());
+    parentCase.setCreatedDateTime(OffsetDateTime.now().minusDays(1));
+    parentCase.setState(null);
+    parentCase.setReceiptReceived(true);
+    parentCase.setRefusalReceived(true);
+    parentCase.setAddressType("HH");
+
+    ResponseManagementEvent managementEvent = getTestResponseManagementEvent();
+    FulfilmentRequestDTO expectedFulfilmentRequest =
+        managementEvent.getPayload().getFulfilmentRequest();
+    expectedFulfilmentRequest.setCaseId(parentCase.getCaseId().toString());
+    expectedFulfilmentRequest.setFulfilmentCode(individualResponseCode);
+
+    when(caseRepository.findByCaseId(UUID.fromString(expectedFulfilmentRequest.getCaseId())))
+        .thenReturn(Optional.of(parentCase));
+
+    // when
+    underTest.processFulfilmentRequest(managementEvent);
+
+    // then
+    verify(eventLogger, times(1))
+        .logFulfilmentRequestedEvent(
+            parentCase,
+            parentCase.getCaseId(),
+            managementEvent.getEvent().getDateTime(),
+            "Fulfilment Request Received",
+            FULFILMENT_REQUESTED,
+            expectedFulfilmentRequest,
+            managementEvent.getEvent());
+
+    ArgumentCaptor<Case> caseArgumentCaptor = ArgumentCaptor.forClass(Case.class);
+    verify(caseRepository).save(caseArgumentCaptor.capture());
+    Case actualChildCase = caseArgumentCaptor.getValue();
+
+    testIndivdualFulfilmentRequestCase(parentCase, actualChildCase);
```

Comment:
probably rename to checkIndivdualFulfilmentRequestCase, as it's not a test per se etc and so on